appveyor.yml

```diff
@@ -41,6 +43,7 @@ install:
   - echo extension=php_intl.dll >> php.ini
   - echo extension=php_mbstring.dll >> php.ini
   - echo extension=php_fileinfo.dll >> php.ini
+  - echo extension=php_xdebug-2.3.3-5.5-vc11-nts.dll >> php.ini
```

Comment:
I though earlier in the day that it could help me better understand what was going on with the TimeHelper tests, but it doesn't. I can surely remove it if you will